samples/zmon-actuator-jetty-jersey/src/test/java/com/zalando/zmon/boot/jetty/BaseJerseyConfigIT.java

```diff
@@ -1,12 +1,12 @@
 /**
  * Copyright (C) 2015 Zalando SE (http://tech.zalando.com)
- *
+ * <p>
```

Comment:
What about removing license headers instead reformatting them ;-)
If I remember correctly they are discouraged by the OpenSource guidelines

Reply:
Has been done on master